firefox/janitor-hg.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Firefox (hg)",
+  "description": "The Firefox browser (hg flavour)",
```

Comment:
Nit: "Mercurial checkout" instead of "hg flavour"? More explicit, and handles any ambiguity between hg/Mercurial